### PR TITLE
evm: remove default addresses from sample.env

### PR DESCRIPTION
**Pull request by gator-boi**

### CODE REVIEW

**Review by RahulMaganti47 — APPROVED** [review]

File: evm/env/.env.sample [hunk 1]
```diff
@@ -25,18 +25,18 @@ export RELEASE_OUTBOUND_LIMIT=0
...
 ### Wormhole Chain ID
-export RELEASE_WORMHOLE_CHAIN_ID=10002
+export RELEASE_WORMHOLE_CHAIN_ID=
 
 ### Wormhole Core Bridge Address
-export RELEASE_CORE_BRIDGE_ADDRESS=0x4a8bc80Ed5a4067f1CCf107057b8270E0cC11A78
+export RELEASE_CORE_BRIDGE_ADDRESS=
 
 ### Wormhole Finality
 export RELEASE_CONSISTENCY_LEVEL=201
```
Comment (djb15) [inline evm/env/.env.sample:34]: Should we remove this default too? Seems dangerous to leave this at 201

**Review by djb15 — COMMENTED** [review]
What about also removing the `RELEASE_OUTBOUND_LIMIT` default? Especially since we currently don't have the tooling to easily update rate limits?

File: evm/env/.env.sample (hunk 1 above)
Comment (djb15) [reply evm/env/.env.sample:34]: Ah wait, is 200 instant, 201 is finalized?

**Comment by gator-boi** [comment]
> What about also removing the `RELEASE_OUTBOUND_LIMIT` default? Especially since we currently don't have the tooling to easily update rate limits?

I added this change to the latest commit. Thanks.

**Review by RahulMaganti47 — APPROVED** [review]

**Pull request merged**